packages/manager/apps/container/src/container/legacy/server-sidebar/universe/DedicatedSidebar.tsx

```diff
@@ -50,6 +50,9 @@ export const features = [
   'network-security',
   'key-management-service',
   'okms:key-management-service',
+  'observability',
+  'observability:dashboards',
+  'observability:metric:tenant',
```

Comment:
Keep consistency -> plural or singular for all
```suggestion
  'observability:dashboards',
  'observability:metrics:tenants',
```